variables.tf

```diff
@@ -326,3 +326,38 @@ variable "public_agents_acm_cert_arn" {
   description = "ACM certifacte to be used for the public agents load balancer"
   default     = ""
 }
+
+variable "dcos_calico_network_cidr" {
```

Comment:
they are already defined in dcos_core_variables.tf defining them again leads to duplicate variable errors

Reply:
Done, thanks.